processing/src/main/java/io/druid/query/aggregation/FilteredAggregatorFactory.java

```diff
@@ -28,7 +28,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-public class FilteredAggregatorFactory implements AggregatorFactory
+public class FilteredAggregatorFactory extends AggregatorFactory
```

Comment:
FilteredAggregatorFactory's combining factory is `delegate.getCombiningFactory()`, so same will never be stored inside metadata . However, changed it to throw unsupported op exception if merging is ever called.